src/extension/editorMessages.ts

```diff
@@ -31,6 +31,12 @@ export namespace EditorMessages {
     export const reportUnexpectedError = (errorDescription: string) =>
         `Coqpilot got an unexpected error: ${errorDescription}. Please report this crash by opening an issue in the Coqpilot GitHub repository.`;
 
+    export const completionAborted =
+        "Completion generation was forcefully aborted, Coq-LSP server stopping. Please hit the CoqPilot Status Bar button again to restart the server and proceed with completions.";
```

Comment:
`Coq-LSP server stopping` -> `stopping Coq-LSP server` or `CoqLSP server is stopping`

Reply:
Also, I feel like `proceed with completions` is a bit ambiguous here: will pressing the button just activate the CoqPlugin again or will it start performing the aborted request again?